### DISCUSSION TITLE
Add support for Fluent message format

### DISCUSSION THREAD

**Discussion by jugaadi** [ember-intl/ember-intl#1617] [Ideas]
[Project Fluent](https://projectfluent.org/) is a family of localization specifications, implementations and good practices developed by Mozilla.

**Comment by longlho** [comment]
Nope because their syntax isn't ICU

**Reply by jugaadi** [reply]
Is it possible to extend ember-intl to add support for Fluent message format?

**Reply by longlho** [reply]
not really, or rather, not easily

**Reply by longlho** [reply]
we're all working on MessageFormat v2 btw, including the Fluent folks https://github.com/unicode-org/message-format-wg/

**Reply by jugaadi** [reply]
Thanks.
